content/scalability/docs/supporting-infrastructure.md

```diff
@@ -0,0 +1,228 @@
+
+```
+ec2-instance-selector --service eks --vcpus-min 8 --memory-min 16
+a1.2xlarge
+a1.4xlarge
+a1.metal
+c4.4xlarge
+c4.8xlarge
+c5.12xlarge
+c5.18xlarge
+c5.24xlarge
+c5.2xlarge
+c5.4xlarge
+c5.9xlarge
+c5.metal
+```
+
+### Prefer larger nodes to reduce control plane load
+
+When deciding what instance types to use, fewer, large nodes will put less load on the Kubernetes Control Plane because there will be fewer kubelets and DaemonSets running. However, large nodes may not be utilized fully like smaller nodes. Node sizes should be evaluated based on your workload availability and scale requirements.
+
+A cluster with three u-24tb1.metal instances (24 TB memory and 448 cores) has 3 kublets, and would be limited to 110 pods per node by default. If your pods use 4 cores each then this might be expected (4 cores x 110 = 440 cores/node). With a 3 node cluster your ability to handle an instance incident would be low because 1 instance outage could impact 1/3 of the cluster. You should specify node requirements and pod spread in your workloads so the Kubernetes scheduler can place workloads properly.
+
+Workloads should define the resources they need and the availability required via taints, tolerations, and [PodTopologySpread](https://kubernetes.io/blog/2020/05/introducing-podtopologyspread/). They should prefer the largest nodes that can be fully utilized and meet availability goals to reduce control plane load, lower operations, and reduce cost.
+
+The Kubernetes Scheduler will automatically try to spread workloads across zones and hosts if resources are available. The Kubernetes Cluster Autoscaler will attempt to add nodes in each Availability Zone evenly. Karpenter will not spread nodes or pods by default. To force workloads to spread you should use topologySpreadConstraints:
+
+```
+spec:
+  topologySpreadConstraints:
+    - maxSkew: 3
+      topologyKey: "topology.kubernetes.io/zone"
+      whenUnsatisfiable: ScheduleAnyway
+      labelSelector:
+        matchLabels:
+          dev: my-deployment
+    - maxSkew: 2
+      topologyKey: "kubernetes.io/hostname"
+      whenUnsatisfiable: ScheduleAnyway
+      labelSelector:
+        matchLabels:
+          dev: my-deployment
+```
+
+### Avoid instances with burstable CPUs
+
+Avoid instance types that use burstable CPUs like T series instances. Workloads should define what size nodes they need to be run on to allow consistent performance and predictable scaling. A workload requesting 500m CPU will perform differently on an instance with 4 cores vs one with 16 cores.
+
+A workload being scheduled in a cluster with Karpenter can use the [supported labels](https://karpenter.sh/docs/concepts/scheduling/#labels) to target specific instances sizes.
```

Comment:
How does this relate to the previous paragraph?

Reply:
This section is about burstable instance, right?

Reply:
Re-worded the title and updated some info to be clearer. This section is about workload performance more than burstable CPU